.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: java
 sudo: false
 script: 
-  - git clone https://github.com/SupunArunoda/CPACEP.git
+  - cd ..
+  - cd ..
+  - cd CPACEP
+  - mkdir /tmp/test
   - mvn clean verify
-  - mvn test
+  - ls /tmp/test
   
-
-before_script:
+before_install:
+  - cd ..
+  - mkdir CPAChecker
+  - cd CPAChecker
   - git clone https://github.com/sosy-lab/cpachecker.git
+  - mkdir /tmp/original
   - cd cpachecker
   - ant build
-  - scripts/cpa.sh -spec config/specification/default.spc -config config/default.properties doc/examples/example.c -outputpath /tmp/cpacep/original -stats
-  
+  - scripts/cpa.sh -spec config/specification/default.spc -config config/default.properties doc/examples/example.c -outputpath /tmp/original -stats
```

Comment:
You actually don't have to change directories to execute CPAchecker. Just use (at the moment) `CPAchecker/cpachecker/scripts/cpa.sh` .

I would also not put this test into the install-phase. If install fails, it will be at the `ant build` step.
Executing cpachecker is more of a "let's see whether it runs properly" check that is independent of installation.

Reply:
This is ok now.